packages/blockchain-link/src/workers/solana/index.ts

```diff
@@ -295,18 +296,34 @@ const getAccountInfo = async (request: Request<MessageTypes.GetAccountInfo>) =>
         } as const;
     }
 
+    // token account's owner is the wallet
+    const getATAOwnerAddress = async (address: Address) => {
```

Comment:
```suggestion
    const getOwnerAddress = async (address: Address) => {
```

can be generic also for other types associated accounts